.github/workflows/update-browser-versions.yml

```diff
@@ -0,0 +1,170 @@
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Check for new Chrome versions
+        id: get-versions
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const https = require('https')
+            const fs = require('fs')
+
+            const getLatestVersionData = () => {
+              const options = {
+                hostname: 'omahaproxy.appspot.com',
+                port: 443,
+                path: '/all.json',
+                method: 'GET'
+              }
+
+              return new Promise((resolve, reject) => {
+                const req = https.request(options, (res) => {
+                  let response = ''
+
+                  res.on('data', (d) => {
+                    response += d.toString()
+                  })
+
+                  res.on('end', () => {
+                    resolve(response)
+                  })
+                })
+
+                req.on('error', (err) => {
+                  reject(err)
+                })
+
+                req.end()
+              })
+            }
+
+            const currentBrowserVersions = JSON.parse(fs.readFileSync('./browser-versions.json'))
+
+            try {
+              const data = JSON.parse(await getLatestVersionData())
+              const linuxData = data.find((item) => item.os === 'linux')
+              const stableData = linuxData.versions.find((version) => version.channel === 'stable')
+              const betaData = linuxData.versions.find((version) => version.channel === 'beta')
+              const hasStableUpdate = currentBrowserVersions['chrome:stable'] !== stableData.version
+              const hasBetaUpdate = currentBrowserVersions['chrome:beta'] !== betaData.version
+              let prTitle = 'Update '
+
+              if (hasStableUpdate) {
+                prTitle += `Chrome (stable) to ${stableData.version}`
+
+                if (hasBetaUpdate) {
+                  prTitle += ' and '
+                }
+              }
+
+              if (hasBetaUpdate) {
+                prTitle += `Chrome (beta) to ${betaData.version}`
+              }
+
+              core.setOutput('has_update', hasStableUpdate || hasBetaUpdate)
+              core.setOutput('current_stable_version', currentBrowserVersions['chrome:stable'])
+              core.setOutput('latest_stable_version', stableData.version)
+              core.setOutput('has_stable_update', hasStableUpdate)
+              core.setOutput('current_beta_version', currentBrowserVersions['chrome:beta'])
+              core.setOutput('latest_beta_version', betaData.version)
+              core.setOutput('has_beta_update', hasBetaUpdate)
+              core.setOutput('pr_title', prTitle)
+            } catch (err) {
+              console.log('Errored checking for new Chrome versions:', err.stack)
+              core.setOutput('has_update', false)
+            }
```

Comment:
it would be nice to move this to `./scripts` so that it can be run manually and so that it can benefit from linting/code highlighting

Reply:
unless there's some reason we can't, i see this `core.setOutput` stuff that i don't recognize...

Reply:
I hesitated a little to do that since there's a bit of gotcha that anything outside of the yml file itself gets run on the branch that's checked out in the action (develop) and not the branch you're working on. If we want to iterate on the action in a separate branch and change the script file, we'll need to temporarily change `env.BASE_BRANCH` to that branch, but I suppose we'd want to do that anyway in the course of working on it and testing it out.